ord_schema/updates_test.py

```diff
@@ -57,7 +58,7 @@ def test_with_updates_simple(self):
     def test_with_no_updates(self):
         message = reaction_pb2.Reaction()
         message.provenance.record_created.time.value = '2020-05-08'
-        message.reaction_id = 'ord-test'
+        message.reaction_id = 'ord-c0bbd41f095a44a78b6221135961d809'
```

Comment:
FYI, this was changed because the update function will only preserve an existing `reaction_id` if it does not appear malformed

Reply:
SGTM